www/docs/customization/gofish.md

```diff
@@ -0,0 +1,120 @@
+---
+title: GoFish
+---
+
+After releasing to GitHub or GitLab, GoReleaser can generate and publish
+a _Fish Food_ Cookbook into a repository that you have access to.
+
+The `rigs` section specifies how the fish food should be created.
+You can check the
+[GoFish documentation](https://gofi.sh/#intro)
+and the
+[Fish food cookbook](https://gofi.sh/#cookbook)
+for more details.
+
+!!! warning
+If you have multiple 32-bit arm versions in each `build` section, and
+you do not specify any `ids` in the rigs section, it will default to all
+artifacts and GoReleaser will fail.
```

Comment:
```suggestion
!!! warning
    If you have multiple 32-bit arm versions in each `build` section, and
    you do not specify any `ids` in the rigs section, it will default to all
    artifacts and GoReleaser will fail.
```

Reply:
done